nicoRDL/dotTheory-JS
# JS-203 (JS-353) Interacting with the DOM: Traversing the DOM

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-353

## Description
- completed tutorials on traversing the DOM using node elements and element relationships
- complete tutorial quizzes

## Steps to Validate
1. Open the index.html
2. Select/make active the input fields
3. For each input field selected, the field should turn YELLOW


https://github.com/nicoRDL/dotTheory-JS/assets/100089185/bc3aea99-1a48-4ed4-b434-33f3cfba18fa


## Risks
N/A
